pkgs/development/python-modules/dateparser/default.nix

```diff
@@ -1,61 +1,54 @@
-{ lib, fetchPypi, buildPythonPackage
-, nose
-, parameterized
-, mock
-, flake8
-, glibcLocales
-, six
-, jdatetime
+{ lib
+, buildPythonPackage
+, isPy3k
+, fetchFromGitHub
 , dateutil
-, umalqurra
 , pytz
-, tzlocal
 , regex
+, tzlocal
+, hijri-converter
+, convertdate
+, parameterized
+, pytestCheckHook
+, GitPython
 , ruamel_yaml
-, python
-, isPy3k
 }:
 
 buildPythonPackage rec {
   pname = "dateparser";
   version = "1.0.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "159cc4e01a593706a15cd4e269a0b3345edf3aef8bf9278a57dac8adf5bf1e4a";
-  };
+  disabled = !isPy3k;
 
-  checkInputs = [
-    flake8
-    nose
-    mock
-    parameterized
-    six
-    glibcLocales
-  ];
-  preCheck =''
-    # skip because of missing convertdate module, which is an extra requirement
-    rm tests/test_jalali.py
-  '';
-
-  checkPhase = ''
-    ${python.interpreter} -m unittest discover -s tests
-  '';
-
-  # Strange
-  # AttributeError: 'module' object has no attribute 'config'
-  doCheck = false;
+  src = fetchFromGitHub {
+    owner = "scrapinghub";
+    repo = "dateparser";
+    rev = "v${version}";
+    sha256 = "0i6ci14lqfsqrmaif57dyilrjbxzmbl98hps1b565gkiy1xqmjhl";
+  };
 
   propagatedBuildInputs = [
     # install_requires
     dateutil pytz regex tzlocal
```

Comment:
Are they supposed to be not under nativeBuildInputs?

Reply:
Why would they if they are listed as install_requires?

Reply:
Most python packages don't get there requires right. If it is not required at runtime it should be build or nativeBuildInputs depending if it is executed. If it is required at runtime this is fine.

Reply:
All four are imported outside of the tests.